include/pros/misc.h

```diff
@@ -723,6 +723,26 @@ double battery_get_capacity(void);
  */
 int32_t usd_is_installed(void);
 
+/**
+ * Lists the files in a directory specified by the path
+ * Puts the list of file names (NOT DIRECTORIES) into the buffer seperated by newlines
+ * 
+ * \note use a path of "\" to list the files in the main directory
```

Comment:
So how does this interact with the typical `\usd\` path prefix to access the root of the SD card?

Reply:
I have not checked to see how it interacted with \usd\, I can check tomorrow. But a path of "\" listed the files in the root directory of the sd card